### PR TITLE
A2 14

### PR DESCRIPTION
**Pull request by lrobert142**
## There will be many things to discuss, so please ask away! ##

- Page for individual staff

- Has a merge from #a2-8, which allows me to use the .reveal class

  - I have 'un-spaghettified' the code so it no longer uses nesting. The reason for this is that with lots of nesting it becomes very difficult to see where you are and maintain the codebase.

- I have moved as many of our project's variables as possible to _custom-settings.scss. This is because I see the _settings.scss file as being ONLY for foundation's stuff. Because if we overwrite one little thing, how do we find it vs everything else foundation has added?

  - Ideally, we end up using variables for anything that should be represented as a number (colors, margins, padding) that way we can change it in one place and everything else changes automatically

- I have split the `components` folder into `components` and `pages`. Components is ONLY for reusable components (has-a). Pages is for specific page styles or component styling altered for a page (is-a)

### CODE REVIEW

**Comment by axrs** [comment]
The `partials` folder is typically designed for `components`. But I see where you're coming from.

File: static/src/assets/scss/_settings.scss [hunk 1]
```diff
@@ -108,7 +107,7 @@ $header-font-family: $body-font-family;
 $header-font-weight: $global-weight-normal;
 $header-font-style: normal;
 $font-family-monospace: Consolas, 'Liberation Mono', Courier, monospace;
-$header-color: inherit;
+$header-color: $black;
```
Comment (axrs) [inline static/src/assets/scss/_settings.scss]: Why change this from inherit to black if it's a Foundation setting?

Comment (axrs) [inline static/src/assets/scss/_settings.scss]: That might have been me too...

File: static/src/pages/staff.html [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+            </ul>
+        </div>
+    </div>
+
+    <div class="large-8 columns staff-details">
+        <h1 class="staff-name">Geoff Toomby</h1>
+        <p class="staff-subtitle">
+            Owner
+        </p>
+        <p class="staff-content">
```
Comment (axrs) [inline static/src/pages/staff.html]: Let's avoid putting classes on each paragraph here. It's going to be difficult for the editor (WYSIWYG) to apply them each time.

Comment (lrobert142) [inline static/src/pages/staff.html]: Good idea!

**Pull request merged**